src/rail_gui.cpp

```diff
@@ -54,8 +54,16 @@ static bool _convert_signal_button;          ///< convert signal button in the s
 static SignalVariant _cur_signal_variant;    ///< set the signal variant (for signal GUI)
 static SignalType _cur_signal_type;          ///< set the signal type (for signal GUI)
 
-/* Map the setting: default_signal_type to the corresponding signal type */
-static const SignalType _default_signal_type[] = {SIGTYPE_NORMAL, SIGTYPE_PBS, SIGTYPE_PBS_ONEWAY};
+/** Settings for which signals are shown by the signal GUI */
```

Comment:
Can you please add dots at the end of sentences? Goes for most of the comments introduced, including the `///<` ones :) Tnx!